### PR TITLE
Submitting model for Lung Segmentation

### PR DESCRIPTION
**Pull request by Rakshith2597**
This contains a model for segmenting lung area in CT Images from LUNA Dataset.
 [Link to paper:](https://ieeexplore.ieee.org/abstract/document/9175649)

### CODE REVIEW

**Comment by vladimir-dudnik** [comment]
First of all, thanks for your intention to contribute to OpenVINO Open Model Zoo. We are appreciate that and want to help your PR went smoothly through contribution review process. We apply certain requirements to the content of Open Model Zoo and expect external contributors will accept and comply with them. The basic requirements to contribution are documented at [CONTRIBUTING.md](https://github.com/openvinotoolkit/open_model_zoo/blob/master/CONTRIBUTING.md) file, it might still not contain all requirements, although we working on improvement of documentation. So, I'll provide comments below.

All models in Open Model Zoo are subject of regular validation which includes conversion to OpenVINO IR with every OpenVINO release, running model and demo accuracy and performance validation. As such, for model contribution, we need to consider validation dataset availability and license. We do not redistribute datasets, but keep and use them internally during regular validation cycles. Obviously, validation dataset license should not conflict with Open Model Zoo license.

Please provide link to LUNA validation dataset, so we may review the license applied and judge if we can accept your PR.
@yssemaev FYI

**Comment by Rakshith2597** [comment]
Link to the dataset: [Linke](https://luna16.grand-challenge.org/Data/) . I have added this link to the lung-segmentation.md file as well. The dataset uses the [Creative Commons Attribution 3.0 Unported License](https://creativecommons.org/licenses/by/3.0/).

**Comment by yssemaev** [comment]
Hi, the dataset: https://luna16.grand-challenge.org/Data/ under Creative Commons Attribution 3.0 Unported License is under review, it will take several weeks, will reply with the decision when done.

File: models/public/Lung_Segmentation/lung-segmentation.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
+# lung-cancer-screening(Multi-stage)
+
+## Use Case and High-Level Description
+
+This is a lung cancer screening composite model which segments the lung region in stage 1. In stage2, model detects the presence of nodules in patches extracted from the previously segmented lung region.
```
Comment (eizamaliev) [inline models/public/Lung_Segmentation/lung-segmentation.md:5]:
```suggestion
This is a lung cancer screening composite model which segments the lung region in stage 1. In stage 2, the model detects the presence of nodules in patches extracted from the previously segmented lung region.
```

File: models/public/Lung_Segmentation/lung-segmentation.md [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+The CNN is defined according to [UNet](https://arxiv.org/abs/1505.04597).
+
+| Metric    | Value        |
+|-----------|--------------|
+| Type      | segmentation |
+| GFlops    | 261.901      |
+| MParams   | 34.526       |
+
+### Performance 
+
```
Comment (eizamaliev) [inline models/public/Lung_Segmentation/lung-segmentation.md:32]:
```suggestion
```

Comment (eizamaliev) [reply models/public/Lung_Segmentation/lung-segmentation.md:32]: obsolete

File: models/public/Lung_Segmentation/lung-segmentation.md [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+### Input
+
+#### Original Model
+
+CT Image, name - 0, shape - `1,1,512,512`, format is `B,C,H,W` where:
```
Comment (eizamaliev) [inline models/public/Lung_Segmentation/lung-segmentation.md:37]:
```suggestion
CT Image, name - `0`, shape - `1,1,512,512`, format is `B,C,H,W` where:
```

File: models/public/Lung_Segmentation/lung-segmentation.md [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+
+- `B` - batch size
+- `C` - channel
+- `H` - height
+- `W` - width
+
+#### Converted Model
+
+CT Image, name - 0, shape - `1,1,512,512`, format is `B,C,H,W` where:
```
Comment (eizamaliev) [inline models/public/Lung_Segmentation/lung-segmentation.md:46]:
```suggestion
CT Image, name - `0`, shape - `1,1,512,512`, format is `B,C,H,W` where:
```

File: models/public/Lung_Segmentation/lung-segmentation.md [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+- `B` - batch size
+- `C` - channel
+- `H` - height
+- `W` - width
+
+## Output
+
+### Original Model
+
+Probabilities of the given pixel to be in the corresponding class, name - 304, shape - 1,1,512,512, output data format is B,C,D,H,W, where:
```
Comment (eizamaliev) [inline models/public/Lung_Segmentation/lung-segmentation.md:57]:
```suggestion
Probabilities of the given pixel to be in the corresponding class, name - `304`, shape - `1,1,512,512`, output data format is `B,C,D,H,W`, where:
```

File: models/public/Lung_Segmentation/lung-segmentation.md [hunk 6]
```diff
@@ -0,0 +1,103 @@
...
+
+B - batch size
+C - channel
+H - height
+W - width
```
Comment (eizamaliev) [inline models/public/Lung_Segmentation/lung-segmentation.md:62]: First, please make bullet list. Sceond, where is `D`?

**Review by eizamaliev — CHANGES_REQUESTED** [review]

File: models/public/Lung_Segmentation/lung-segmentation.md (hunk 6 above)
Comment (Rakshith2597) [reply models/public/Lung_Segmentation/lung-segmentation.md]: 1. Bullet list has been added.
2. I have removed `D`. It was a typo.

**Pull request closed without merging**